### ISSUE TITLE
Catalina: New install of SN app won't open in Catalina 

### ISSUE THREAD

**Issue by krgp21** [standardnotes/forum#715]
"The application “Standard Notes.app” can’t be opened."

Other users having same problem in Reddit. Am using 10.15 Beta (19A582a).

Thanks.

**Comment by krgp21** [comment]
Update: I downloaded .15 from GH and it launched fine. It's .16 that seems to be the problem.

**Comment by ebaines3** [comment]
I experienced the same thing. Version 3.0.16 of SN would not open a fresh on a fresh install of the release version of Catalina, yet version 3.0.15 does open.

**Comment by sschultze** [comment]
I experienced the same with version 3.0.17. Does not open. Downloaded and installed version 3.0.15 and everything works fine.

**Comment by mobitar** [comment]
Hoping to investigate this sometime this week or early next.

**Comment by ebaines3** [comment]
If it's helpful, I've had this same issue with a few other Electron apps (Signal, for instance). You may already be beyond this point, but here's a link to a fix that may work. Thank you so much for all your work!

https://medium.com/cacher-app/getting-your-electron-app-working-on-macos-catalina-10-15-63e53f397da2

**Comment by mobitar** [comment]
Nice find, thanks for the resource! This may come in handy.

**Comment by mobitar** [comment]
Related thread: https://github.com/standardnotes/desktop/issues/399#issuecomment-541628451

One user said: 

> Scratch that, installing 3.0.17 from the [github releases page](https://github.com/standardnotes/desktop/releases) fixed it for me. Not sure how that differs from the version on standardnotes..org.

Not sure if anyone can replicate. Downloads from our site are also served from GitHub, so shouldn't really make a difference.

**Comment by AlexRowanCom** [comment]
Here is a way of getting the SN zip to work:
Download zip file.
Install the free "The Unarchiver" from the app store.
Right click file use "Open with" to extract by selecting "The Unarchiver".
Click on Standard Notes.
Approve macOS security routine.
It works.
I'm no tech expert so am ignorant of many things - it does seem to me however that the problem is at least in part with Apple's own Archive Utility (which can also struggle with large zip files). I have set "The Unarchiver" as my default - I have not tested whether extracting SN via a double click will work - I decided to use the 'Open With' method.
